src/merger/merger.c

```diff
@@ -1256,15 +1267,17 @@ luaopen_tuple_merger(struct lua_State *L)
 	CTID_STRUCT_TUPLE_MERGE_SOURCE_REF =
 		luaL_ctypeid(L, "struct tuple_merge_source&");
 
-	/* Export C functions to Lua. */
+	/* Create the module table. */
 	static const struct luaL_Reg meta[] = {
 		{"new_buffer_source", lbox_merger_new_buffer_source},
 		{"new_table_source", lbox_merger_new_table_source},
 		{"new_tuple_source", lbox_merger_new_tuple_source},
 		{"new", lbox_merger_new},
 		{NULL, NULL}
 	};
-	luaL_register(L, "tuple.merger", meta);
+	size_t len = sizeof(meta) / sizeof(meta[0]);
+	lua_createtable(L, 0, len);
+	luaL_register(L, NULL, meta);
```

Comment:
It'll count `{NULL, NULL}`, so we should substract 1.

Reply:
Yes, thanks. Fixed.